resources/schema.py

```diff
@@ -178,8 +189,7 @@ class Meta:
         interfaces = (relay.Node,)
         filterset_class = BerthNodeFilterSet
 
-    @login_required
-    @superuser_required
+    @view_permission_required(BerthLease, BerthApplication, BerthLease)
```

Comment:
second `BerthLease` should probably be `CustomerProfile` 👀

Reply:
:derp: good catch, again.